profiles-paper-gui/src/main/java/com/awooga/profiles/dao/impl/PlayerProfilesDAOImpl.java

```diff
@@ -118,14 +119,15 @@ public boolean deleteProfile(Player player, UUID profileUuid) {
 	@SneakyThrows
 	@Override
 	public List<@NotNull ProfileEntity> getProfileEntitiesByGenuineUUID(UUID genuineUuid) {
-		PreparedStatement stmt = conn.prepareStatement("SELECT * FROM profiles WHERE playerUuid=? AND deleted=false");
+		PreparedStatement stmt = conn.prepareStatement("SELECT * FROM profiles WHERE playerUuid=? AND deleted=false"
+			, ResultSet.TYPE_SCROLL_SENSITIVE, ResultSet.CONCUR_READ_ONLY);
```

Comment:
There are other prepared statements in this file. What is the root cause here, and should this fix be applied proactively to the other statements?

Reply:
What was happening here is that you were grabbing an id from the ResultSet outside of while(resultSet.next()).
I do not think that it is causing an issue now, since one of the other patches removed the part of code reading from the resultSet outside of the while statement.
Basically, if you are accessing a ResultSet in any way other than forward (with resultSet.next()) you need to give it a type other than TYPE_FORWARD_ONLY.